pkg/virt-controller/watch/vm.go

```diff
@@ -2637,14 +2637,15 @@ func (c *VMController) sync(vm *virtv1.VirtualMachine, vmi *virtv1.VirtualMachin
 		}
 
 		if syncErr == nil {
+			var vmUpdateErr error
```

Comment:
I do not see a good reasoning to have this commit.

Reply:
>  If it bothers you a lot, you can rename the errors in a separate commit.
https://github.com/kubevirt/kubevirt/pull/9896#discussion_r1250972809
It bothers me.

Reply:
This commit changes more than the name.
(line 2640 and 2648 is not required)